azext_iot/operations/hub.py

```diff
@@ -1611,6 +1611,7 @@ def iot_get_sas_token(
     cmd,
     hub_name=None,
     device_id=None,
+    device_connection_string=None,
```

Comment:
Instead of supporting an offline mode just for device connection strings, I'm thinking it would be valuable to support either hub, device or module offline SAS token  generation via generic  `connection_string` corresponding to a new `--connection-string` argument on this command.

Reply:
@digimaun absolutely agree. This just happened to cover my use case for IoT Central but agree that there's no reason to restrict to only device connection strings